src/FormatNumber/Locales.elm

```diff
@@ -44,6 +45,18 @@ type Decimals
     | Max Int
     | Exact Int
 
+{-| The 'NumericSystem' type contains different numeric systems currently 
+supported:
+
+  - Western - It separates digits by 1000s. 1000000 -> 1,000,000
+  - Indian - It separates digits by 1000 then it is separated by 100s. 
+    1000000 -> 10,00,000
```

Comment:
For consistency we could use the same formatting and style as in `Decimals`:

```suggestion
  - `Western` separates digits by thousands (e.g.: 1000000 might be separated
    as 1,000,000).
  - `Indian` separates digits by the thousand and, from thaere, bu hundreds
    (e.g.: 1000000 might be separated as 10,00,000).

```

Reply:
I will the change in next commit from my fork.